public/app-download.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>App Download</title>
+    <meta name="viewport" content="width=device-width,user-scalable=no,initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,300italic,500" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="css/gallery.build.css">
+    <link rel="stylesheet" href="css/gallery.theme.css">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    
+    <section class="download-app-section">
+      <div class="download-img">
+        <img src="assets/backgroundBlueIphone.png" alt="">
```

Comment:
This is broken in "production" (looks like the server 404's). Works locally, so it's probably a path-related issue.

Additionally, your naming here is a little inconsistent -- you have `backgroundBlueIphone.png` for this, but simply `htcPhone.png` for the other.